controllers/phapplication_controller.go

```diff
@@ -28,6 +28,7 @@ type PhApplicationReconciler struct {
 	PrimeHubCache *phcache.PrimeHubCache
 	PhfsEnabled   bool
 	PhfsPVC       string
+	PrimehubUrl   string
```

Comment:
Can we use `PrimeHubURL` instead to make it consistent with `phfsPVC` & `PrimeHubCache` above?